src/calculateCellHeight.js

```diff
@@ -21,9 +20,5 @@ export default (value, columnWidth, useWrapWord = false) => {
     throw new Error('Column width must be greater than 0.');
   }
 
-  if (useWrapWord) {
-    return wrapWord(value, columnWidth).length;
-  }
-
-  return Math.ceil(stringWidth(value) / columnWidth);
+  return wrapCell(value, columnWidth, useWrapWord).length;
```

Comment:
Is length safe to use here?

Does it handle ansi?

I think not.

I think you need to use `string-width`.

Please add a test case with ansi characters.

Reply:
This is an array, so I would imagine `.length` is safe to use.

I will add the test case. What are you looking for, ANSI-wise?

Reply:
All good – misunderstood the change.

Thank you for adding the test case.